lib/beaker/dsl/install_utils/foss_utils.rb

```diff
@@ -689,23 +689,38 @@ def install_puppet_from_dmg_on( hosts, opts )
         # @return nil
         # @api private
         def install_puppet_agent_from_dmg_on(hosts, opts)
+          require 'nokogiri'
```

Comment:
@johnduarte have you guys seen [what PE does](https://github.com/puppetlabs/beaker-pe/blob/4fc88d8c83bce01ec9a918d3dfdf7e8496e8fc49/lib/beaker-pe/options/pe_version_scraper.rb) for this?

The latest file is usually just a version for them, are you trying to get additional information from there?

If so, I wonder if potentially having that file be a JSON or ruby hash would be a simpler method of providing these values, rather than parsing HTML.  We'd prefer to limit our exposure to Nokogiri if possible.